src/BiometryService/BiometryOptions.cs

```diff
@@ -0,0 +1,51 @@
+namespace BiometryService
+{
+	/// <summary>
+	///     Options for configuring application behaviour when using biometrics.
+	/// </summary>
+	public class BiometryOptions
```

Comment:
Should the option to offer to use the pincode as fallback be offered in this class?